src/bufr/BufrParser/Query/ResultSet.cpp

```diff
@@ -198,6 +198,12 @@ namespace bufr {
                     metaData->rawDims[pathIdx] = maxCount;
                 }
 
+                if (target->exportDimIdxs.size() - 1 < exportIdxIdx)
```

Comment:
```suggestion
                if ( static_cast<int>(target->exportDimIdxs.size() - 1) < exportIdxIdx)
```

Reply:
I doubt this will change much but it gets rid of compiler warning.

Reply:
@rmclaren do you mind if we commit this?  

we are rebuilding the CDASH server right now may be good to wait for that to complete and can use this commit to test it.  Do you mind if we commit this at the end of the day today to test both this change and the CI?

Reply:
@rmclaren @BenjaminRuston This still failing for gnu and clang. It's a bit difficult to see it since the CI is broken.